flink-formats/flink-json/src/test/java/org/apache/flink/formats/json/JsonRowDeserializationSchemaTest.java

```diff
@@ -61,19 +65,21 @@ public void testTypeInfoDeserialization() throws Exception {
 		root.put("id", id);
 		root.put("name", name);
 		root.put("bytes", bytes);
+		root.putObject("map").put("flink", 123);
 
 		byte[] serializedJson = objectMapper.writeValueAsBytes(root);
 
 		JsonRowDeserializationSchema deserializationSchema = new JsonRowDeserializationSchema.Builder(
 			Types.ROW_NAMED(
-				new String[]{"id", "name", "bytes"},
-				Types.LONG, Types.STRING, Types.PRIMITIVE_ARRAY(Types.BYTE))
+				new String[]{"id", "name", "bytes", "map"},
+				Types.LONG, Types.STRING, Types.PRIMITIVE_ARRAY(Types.BYTE), Types.MAP(Types.STRING, Types.LONG))
```

Comment:
Thanks @libenchao ,basically good, do you think we can add a test case with nested map type, say the map value is also a map.

Reply:
good idea, will add it later.

Reply:
done.